composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name":        "ezsystems/ezpublish-legacy",
+    "name":        "sport1online/ezpublish-legacy",
```

Comment:
mmm, sorry, i don't think ezsystems would like to change this :)
even now i realized this pr is closed... is it closed in favour or any other?